rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
So does the bracket syntax work? I could add this feature to the generator if so, then we can handle the `USE` flags on an as-needed basis.